docs/contribute.rst

```diff
@@ -351,6 +351,25 @@ good to include respective documentation. You can add or update
 the :ref:`specification`, extend the :ref:`examples` or write a
 new chapter for the user :ref:`guide`.
 
+tmt documentation is written with `reStructuredText`__ and built
+with `Sphinx`__. Various features of both reST and Sphinx are used
+widely in tmt documentation, from inline markup to references. Feel
+free to use them as well to link new or updated documentation to relevant
+parts, to highlight important points, or to provide helpful examples.
+
+A couple of best practices when updating documentation:
+
+* when referring to a plugin, its options or documentation, prefer
+  reference to ``plugin/STEP/PLUGIN`` rather than to
+  ``/spec/plans/STEP/PLUGIN``. ``:ref:`plugin/prepare/ansible``` is
+  better than ``:ref:`/spec/plans/prepare/ansible```.
```

Comment:
If you check the documentation, it doesnt look as it should. If you remove the double \`, it will fix it but even after fixing it, I dont know if we want to look like this: ... `ansible` is better than `ansible`. Maybe adding some text link to make it more readable?

Reply:
I was worried it might get rendered as a mess, but it seemed good on my end. I will try a `code-block`, that should be safe.

Reply:
Hmm it rendered fine for me

Reply:
Okay, I wanted to make sure if you are okay that the ref links are not clickable

Reply:
Does it rendered fine in this [doc](https://tmt--3007.org.readthedocs.build/en/3007/contribute.html#docs)

Reply:
ah, I understand now. This is an example how to write docs, so I believe this is how it supposed to look, yeah.